pkg/vcs/git.go

```diff
@@ -672,3 +567,144 @@ func ParseGitDiff(patch []byte) []string {
 	}
 	return files
 }
+
+type Git struct {
```

Comment:
Why public? Because following PRs need it?

Reply:
Got it now. We have a git command line tool wrapper and the git repo abstraction now.